### PR TITLE
[Backport 2.x] Merge pull request #468

### PR DESCRIPTION
**Pull request by TackAdam**
### Description
[Describe what this change achieves]
cherrypick backport for Cypress upgrade to v12.8.1

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -1,26 +0,0 @@
-{
```
Comment (derek-ho) [inline cypress.json:1]: why was this deleted?

File: cypress.config.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  fixturesFolder: '.cypress/fixtures',
+  screenshotsFolder: '.cypress/screenshots',
+  videosFolder: '.cypress/videos',
+  downloadsFolder: '.cypress/downloads',
+  viewportWidth: 2000,
+  viewportHeight: 1320,
+  requestTimeout: 60000,
+  responseTimeout: 60000,
+  defaultCommandTimeout: 60000,
+  //experimentalNetworkStubbing: true,
```
Comment (mengweieric) [inline cypress.config.js:15]: Do we need these settings, we can delete if we don't need

**Review by mengweieric — APPROVED** [review]

File: cypress.config.js (hunk 2 above)
Comment (TackAdam) [reply cypress.config.js:15]: They are the recommended fixes to a chrome crashing error I have thoroughly tested https://docs.cypress.io/guides/references/error-messages#The-Chromium-Renderer-process-just-crashed.
Leaving them in to indicate that it has been tried but was not effective. If we end up bumping the version in the future these might be used but lowering the test kept in memory has been the only effective fix. I think keeping them in the file as settings that can be turned on might help in the future.

**Review by ps48 — APPROVED** [review]
